packages/ui/src/components/banner-container/banner-container.tsx

```diff
@@ -1,9 +1,11 @@
 import {Component, h} from 'preact';
 import {ContribLogger, getContribLogger} from '@playkit-js-contrib/common';
 import * as styles from './_banner-container.scss';
+import BannerConfig = KalturaPlayerContribTypes.BannerConfig;
 
 export interface BannerContainerProps {
   onClose: () => void;
+  bannerConfig: BannerConfig;
```

Comment:
consider asking specific the `theme` if possible, to make the public API of the component better

Reply:
done